.gitlab-ci.yml

```diff
@@ -463,11 +461,13 @@ CleanUpReview:
   environment:
     name: development
     action: stop
-    kubernetes:
-      namespace: ${CI_COMMIT_REF_SLUG}
+  variables:
+    AGENT: ${DEV_AGENT}
+    DEV_NAMESPACE: ${DEV_NS}
+    NAMESPACE: ${CI_COMMIT_REF_SLUG}
   except:
-  - dev
-  - master
+    - dev
+    - main
```

Comment:
ensembl-client doesn't have a `main` branch; it still has a `master`

Reply:
Fixed